### DISCUSSION TITLE
Which tool can convert .safetensors to .ckpt

### DISCUSSION THREAD

**Discussion by sugoidesune** [diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI#24] [Q&A]
I have some older pipeline working and don't have the ability yet to update it.
Is there something out there?

**Comment by danielgangl** [comment]
@diStyApps having the same issue - been searching everywhere for some time. Can you maybe help?

**Reply by diStyApps** [reply]
What is the issue?

**Reply by danielgangl** [reply]
How to convert .safetensors to .ckpt. Is this even possible?
